### ISSUE TITLE
Missing Interactive Elements in leaflet.html in Google Chrome Browser

### ISSUE THREAD

**Issue by catharinadepari** [jakobzhao/geog458#14]
Hello everyone,

I have an issue with displaying the interactive elements in my leaflet project (index.html file) from google chrome browser. However, I found the leaflet worked properly with interactive elements when launching the Atom’s live server (http://127.0.0.1:3000/catharinaleaflet/usairport.html).

Here are the errors I found from the google inspection:

1. Access to XMLHttpRequest at 'file:///C:/WorkspaceGEOG458/catharinaleaflet/assets/us-states.geojson' from origin 'null' has been blocked by CORS policy: Cross origin requests are only supported for protocol schemes: http, data, chrome, chrome-extension, https.
2. Failed to load resource: net::ERR_FAILED
3. Failed to load resource: the server responded with a status of 400 ()

My questions are:
1.	How can we ensure that the leaflet/html file could be accessed similarly to what is shown in the Atom’s live server by other users through google chrome browser? 
2.	Can we rely only on Atom’s live server and assume that other users can see the same file through their browsers? 

I have read many suggestions, one of them from this link: https://stackoverflow.com/questions/20041656/xmlhttprequest-cannot-load-file-cross-origin-requests-are-only-supported-for-ht. But I am not sure which one I should follow except perhaps the idea that we can just check through our local server (Atom’s liver server).

Thanks in advance. I really appreciate any kinds of suggestions from you.

**Comment by jakobzhao** [comment]
could you please share the repo with us in order to better identify the bug?

**Comment by catharinadepari** [comment]
Sure, Prof Zhao. Here is my repo address:

https://github.com/catharinadepari/catharinaleaflet

I also unfortunately found another issue, blob/master integrated to my url when accessing the index.html : https://github.com/catharinadepari/catharinaleaflet/blob/master/index.html

Thank you in advance, Prof Zhao.

**Comment by jouho** [comment]
Hi @catharinadepari, 

I did not have such issues when I submitted this assignment, but I had the same problem when I try to open my HTML file today. I suspect that there was some sort of update or change made to the chrome. My GitHub page somehow still works, so I recommend you try deploying your HTML on GitHub pages. Here is my repo for reference: https://github.com/jouho/us_airports_map.github.io

**Comment by jakobzhao** [comment]
@catharinadepari apparently, you can access a local path (c://) from a server (127.0.0.1:300). Please use a relative path of the geojson file instead of the absolute local path
